osu.Game/Screens/SelectV2/BeatmapMetadataWedge.cs

```diff
@@ -298,7 +299,11 @@ private void updateDisplay()
             else
                 source.Data = ("-", null);
 
-            mapperTags.Tags = (metadata.Tags.Split(' '), t => songSelect?.Search(t));
+            if (!string.IsNullOrEmpty(metadata.Tags))
+                mapperTags.Tags = (metadata.Tags.Split(' '), t => songSelect?.Search(t));
+            else
+                mapperTags.Tags = (Array.Empty<string>(), _ => { });
```

Comment:
Try not to look too much into how weird this code reads, it's refactored in https://github.com/ppy/osu/pull/33687 to look more sane than it is right now.